core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -61,19 +86,51 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
-      LOG.debug("Internal Alluxio error when running rpc", e);
+      LOG.debug("Internal Error: {}", callable, e);
+      throw e.toThrift();
+    } catch (IOException e) {
+      LOG.debug("I/O Error: {}", callable, e);
```

Comment:
I prefer having the category associated with the exception, so the user knows if there is something wrong within Alluxio, with I/O, or if something very bad has happened (unexpected). This information is hard to gather from just looking at the exception.